enumerables.rb

```diff
@@ -0,0 +1,135 @@
+# rubocop: disable Metrics/ModuleLength, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
+module Enumerable
+  def my_each
```

Comment:
Enumerable each method calls the given block once for each element in self, passing that element as a parameter. Returns the array itself.

![Screen Shot 2020-06-27 at 2 45 36 PM](https://user-images.githubusercontent.com/52430951/85930808-e24b4900-b884-11ea-917f-1560cdbf4644.png)